cli/main.rs

```diff
@@ -44,12 +44,14 @@ use std::env;
 use std::iter::once;
 use std::path::PathBuf;
 
-fn init_v8_flags(v8_flags: &[String]) {
-  let v8_flags_includes_help = v8_flags
+fn init_v8_flags(v8_flags: &[String], env_v8_flags: Vec<String>) {
+  let v8_flags_includes_help = env_v8_flags
     .iter()
+    .chain(v8_flags)
     .any(|flag| flag == "-help" || flag == "--help");
   // Keep in sync with `standalone.rs`.
```

Comment:
Maybe this code could be extract out somewhere common (util dir maybe) then shared with standalone.rs? standalone.rs could then provide an empty vector for the env_v8_flags so it's clear that part of the code doesn't read the environment.

Reply:
done